src/js/utils/DOM.js

```diff
@@ -64,18 +64,28 @@ export const containsFocus = node => {
   return !!element;
 };
 
-export const getFirstFocusableDescendant = element => {
+export const isFocusable = (element) => {
+  const tagName = element.tagName.toLowerCase();
+  return tagName === 'input' || tagName === 'select';
```

Comment:
This feels a bit fragile to me. For instance, it doesn't cover `<textarea />`.

Reply:
I agree, I just got this logic from the getFirstFocusableDescendant and split in a different method.
I can add `<textarea />` do you think need to create for some different element? 
Or I can create a different method, like:
```
export const checkTagName = (element, tagList=["input", "select", "textarea"]) => {
  const tagName = element.tagName.toLowerCase();
  return tagList.includes(tagName);
})
```
What do you think?

Reply:
Ah, I didn't realize this was just being copied. I'm OK with just adding "textarea" for this pull request.